src/profile_manager/views.py

```diff
@@ -24,10 +24,22 @@
 from django.contrib.auth.forms import SetPasswordForm
 
 
+class ViewTypes:
```

Comment:
I like this.  Could probably be used to good effect in other areas that hack this by looking at the url. Can you add a new issue to "Find and refactor templates that look at the url for setting active buttons" and provide this example for how to do it

Reply:
#1150 added